api/api/spec/spec.yaml

```diff
@@ -208,43 +247,32 @@ components:
       - pending
       - neverconnected
       - disconnected
+      description: Agent status. It is calculated based on the last keep alive and the Wazuh version.
+
+    AgentID:
+      type: string
+      minLength: 3
+      description: Agent ID.
 
     AgentConfiguration:
       type: object
-      required:
-        - configuration
+      description: Configuration the agent is currently using. The output of this API call depends on the component requested and the agent configuration.
+
+    CDBList:
+      type: object
       properties:
-        configuration:
+        items:
           type: array
           items:
-            required:
```

Comment:
this required key has been lost

Reply:
That required was part of the `AgentConfiguration` model. This model depends on the user configuration and, therefore, it can't be specified in the spec.